05 - VerySimpleFixedPVC2_with_electrical_layout.pvc

```diff
@@ -321,6 +318,9 @@
 			</node>
 			<!-- We instantiate a first fixed tilt table, without any transformation -->
 			<node id="PVnode1" name="PVnode1" sid="PVnode1">
+				<rotate sid="azimuth">0 0 1 45</rotate>
+				<rotate sid="base slope">0 1 0 10</rotate>
+				<rotate sid="tilt">1 0 0 21.8</rotate>

```

Comment:
`tilt`, `base_slope` and `azimuth` are regular transformations.
This way, they are understood by a non-pv 3d software